spec.md

```diff
@@ -137,6 +137,28 @@ attributes and the event data will be materialized. For example, in the case
 of a JSON serialization, the context attributes and the event data might
 both appear within the same JSON object.
 
+### Extension Attributes
+CloudEvent producers MAY include additional extension attributes within the
+event. This enables event producers, or middleware, to include additional
+metadata that might be used for any purpose in the processing of the
+CloudEvent, such as identifying or correlating event sources. See
+[CloudEvent Attributes Extensions](primer.md#cloudevent-attribute-extensions)
+for additional information concerning the use and definition of
+extensions.
+
+This specification places no restriction on the type or semantics of the
+extension attributes. Each definition of an extensions SHOULD fully
+define all aspects of the attribute - e.g. its name, semantic meaning
+and possible values.  New extension definitions SHOULD use a name that is
```

Comment:
For some metadata that is in the format of property/classification bag, we should only define the name of the bag and the format of the bag, but leave the values inside the bag to be customizable and added by the event producer. Suggest to remove "Each definition of an extensions SHOULD fully define all aspects of the attribute - e.g. its name, semantic meaning and possible values. ". 

To avoid name collision,  property will be used by a large number of use cases, then the WG should define a consistent name for that property bag in the main spec.md.

Reply:
I tried to clarify this.

Reply:
@duglin Where can I see your new texts and clarifications. Are you going to post a new diff?